bindings/generator/api/client.py

```diff
@@ -149,3 +153,21 @@ async def client_workspace_share(
     role: Optional[RealmRole],
 ) -> Result[None, ClientWorkspaceShareError]:
     raise NotImplementedError
+
+
+class ClientWorkspaceInfoError(ErrorVariant):
+    class Internal:
+        pass
+
+
+async def client_info(
+    client: Handle,
+) -> Result[
+    Tuple[OrganizationID, DeviceID, UserID, UserProfile, Optional[HumanHandle]],
+    ClientWorkspaceInfoError,
+]:
+    raise NotImplementedError
+
+
+async def client_display(client: Handle) -> Result[Tuple[str, str], ClientWorkspaceInfoError]:
```

Comment:
Can we create a struct here ? I don't know what correspond the first and second str

Reply:
Superseded by https://github.com/Scille/parsec-cloud/pull/5198#discussion_r1324289056

Reply:
Outdated